tests/packages/install-subdir-exclude/meson.build

```diff
@@ -0,0 +1,45 @@
+# SPDX-FileCopyrightText: 2023 The meson-python developers
+#
+# SPDX-License-Identifier: MIT
+
+project('install-subdir-exclude', version: '1.0.0')
+
+py = import('python').find_installation()
+purelib = py.get_install_dir(pure: true)
+
+# We need two foo directories because of a bug in the Meson metadata generation.
+# https://github.com/mesonbuild/meson/issues/11539
+
+# Examples removed from https://mesonbuild.com/Reference-manual_functions.html#install_subdir
+install_subdir(
+  'foo',
+  install_dir: purelib / 'subdir1',
+  strip_directory : false,
+)
```

Comment:
An empty line between `install_subdir()` calls makes reading this a bit easier.

Reply:
Sure.